code/datums/diseases/advance/symptoms/weight.dm

```diff
@@ -27,7 +27,7 @@ Bonus
 /datum/symptom/weight_gain/Activate(var/datum/disease/advance/A)
 	..()
 	if(prob(SYMPTOM_ACTIVATION_PROB))
-		var/mob/living/M = A.affected_mob
```

Comment:
you dont need to change this

Reply:
you could even just directly reference A.affected_mob since thats already carbon

Reply:
This code is in for removal. No need to waste time on improving it.